### PR TITLE
Update readme note to more strongly emphasize URL scheme configuration

### PR DESCRIPTION
**Pull request by iujames**
Trying out the new 

> [!IMPORTANT] important note callout

[reference](https://blog.jakelee.co.uk/github-alert-experiments/)

just to see what that looks like here...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,9 @@ Initializing the SDK requires you to provide two values, an Appcues account ID,
 
 #### Supporting Debugging and Experience Previewing
 
-Supporting debugging and experience previewing is not required for the Appcues React Native Module to function, but it is necessary for the optimal Appcues builder experience. Refer to the [URL Scheme Configuration Guide](https://github.com/appcues/appcues-react-native-module/blob/main/docs/URLSchemeConfiguring.md) for details on how to configure.
+During installation, follow the steps outlined in [Configuring the Appcues URL Scheme](https://github.com/appcues/appcues-react-native-module/blob/main/docs/URLSchemeConfiguring.md). 
+
+> [!IMPORTANT] This is necessary for the optimal Appcues builder experience, to support experience preview, screen capture and debugging.
```
Comment (mmaatttt) [inline README.md:79]: I think you need a newline after `[!IMPORTANT]`? https://blog.jakelee.co.uk/github-alert-experiments/#syntax

Comment (mmaatttt) [reply README.md:79]: And maybe instead of "optimal" say "complete"? It's pretty much required, especially with screen capture/anchored tooltips.

Comment (iujames) [reply README.md:79]: yep, weird, I force pushed an update to this branch but the PR keeps showing me the older commit here for some reason.

Comment (iujames) [reply README.md]: ok, now its updated.

Screenshot 2023-07-31 at 11 56 16 AM

**Review by mmaatttt — APPROVED** [review]
Works for me. Honestly, in the context of the whole doc, the URL scheme stuff being flagged as important seems a bit like overkill, but no objections if it'll help people get it right.

**Comment by iujames** [comment]
> the URL scheme stuff being flagged as important seems a bit like overkill

Really, I agree. I wanted to try the new block just because I saw that, but I don't really think it should be used in this one spot, in retrospect. The most important thing was just making the wording slightly more clear (here and in all the others copied from it), so I'm going to just focus on that.

**Comment by iujames** [comment]
@mmaatttt appreciate your patience working through the options! want to get to a solid wording that we can replicate across the other repos. Pushed up the latest update  - renaming the section header and removing the !IMPORTANT callout. How's that look to you?

**Review by mmaatttt — APPROVED** [review]
Seems clear to me!

**Review by andretortolano — APPROVED** [review]

**Comment by andretortolano** [comment]
this one is ready to merge right

**Pull request merged**